scripts/run-monitor-all.ps1

```
@@ -0,0 +1,88 @@
+# 🏥 System Health Monitoring Script (PowerShell)
+$date = Get-Date -Format "yyyy-MM-dd HH:mm:ss"
+$reportPath = "SYSTEM_HEALTH.md"
+
+Write-Output "# 🏥 System Health Report" | Out-File -FilePath $reportPath -Encoding utf8
+Write-Output "**Date:** $date`n" | Out-File -Append -FilePath $reportPath -Encoding utf8
+
+# Helper to run a monitor and save JSON
+function Run-Monitor {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Run-Monitor' uses an unapproved verb.

(PSUseApprovedVerbs)